docker/stacks/stacks-regtest-miner.toml

```diff
@@ -43,28 +59,23 @@ auth_token = "12345"
 # Add stacks-api as an event observer
 [[events_observer]]
 endpoint = "stacks-api:3700"
-retry_count = 255
-include_data_events = false
```

Comment:
qq: were those deprecated?

Reply:
not even deprecated, hard config error now